edgelet/contrib/systemd/debian/iotedge.service

```diff
@@ -11,6 +11,7 @@ KillMode=process
 TimeoutStartSec=600
 TimeoutStopSec=40
 Restart=on-failure
+RestartPreventExitStatus=153
```

Comment:
Does this still require a `systemctl reset-failed + systemctl start` to start it back up? Or is just `systemctl start` sufficient?

Reply:
Just `systemctl start`. You can call `systemctl reset-failed` to change the unit's state from `failed` to `inactive`, but since there's no rate limiting in effect it's not necessary.